### DISCUSSION TITLE
Support for name.com dns challenge

### DISCUSSION THREAD

**Discussion by Mrdindon** [NginxProxyManager/nginx-proxy-manager#3739] [Ideas]
Had to swap from godaddy to name.com since they don't support api access for small clients anymore (!).
I migrated to name.com without checking if npm was supporting them for dns challenge with letsencrypt and it's not the case.
Would it be hard to implement it or letting me know how to add it to the list of dns providers ?

**Comment by scottyan19** [comment]
have same question , how to add it manually ?

**Comment by tigins** [comment]
I really don't understand why name.com isn't supported when it's so widespread. What's the reason?

**Comment by Fratopolis** [comment]
Same here, Been using the free cert they provide but that does not cover sub domains. Been doing this for several years.

**Comment by Stephan-P** [comment]
In addition to support for yet another DNS provider, would it be feasible to add support for a custom DNS challenge setup?

**Comment by arcimus** [comment]
Please implement name.com!

**Comment by samwathegreat** [comment]
@arcimus @Stephan-P @Fratopolis @tigins @scottyan19 @Mrdindon

NginxProxyManager uses Certbot for certificates, so the appropriate place to ask for implementing a new provider is there:  https://github.com/certbot/certbot

Good Luck!!

**Comment by jc21** [comment]
Umm what about https://pypi.org/project/certbot-dns-namecom/ ?

BTW `Name.com` is available in the dns providers list.

<img width="589" height="530" alt="Screenshot 2026-06-08 at 8 54 37 am" src="https://github.com/user-attachments/assets/d4f47cee-e2b7-45ad-8458-a026100a1aa5" />

Closing.
